### PR TITLE
DOCS-2717: Disable config resources

### PR DESCRIPTION
**Pull request by sguequierre**
* Fix error from picture-in-picture pr where info was only in components section and not both
* Add information about disabling component or service

### CODE REVIEW

File: docs/fleet/provision.md [hunk 1]
```diff
@@ -12,10 +12,11 @@ aliases:
   - "/build/provision/"
 ---
 
-{{<gif webm_src="/platform/provisioning-demo.webm" mp4_src="/platform/provisioning-demo.mp4" alt="Using the Viam mobile app to provision a new machine with the Viam Agent." class="alignright" max-width="400px">}}
+<!-- Generally: I think this page can be split up into 3 themes – what provisioning is and how end users would use it, how to configure it, and the nitty gritty details. -->
+
+You can provision a machine, giving it a pre-defined configuration as it first comes online, with Viam's software provisioning manager, the _Viam Agent_.
```
Comment (npentrel) [inline docs/fleet/provision.md:17]: I think that wasn't meant to be here

Comment (sguequierre) [reply docs/fleet/provision.md:17]: oh oops sorry, need to drop this commit

File: docs/configure/_index.md [hunk 2]
```diff
@@ -210,6 +213,18 @@ You must configure a service with a `name` and a `type`:
 The other aspects of configuring a service are highly specific to the type of service.
 See the [services documentation](/services/) for more information.
 
+Some resources have a **TEST** section on the bottom half of their configuration pane which you can expand and interact with to test out controlling the service.
+You must be running `viam-server` and connected to your machine to use this feature.
+
```
Comment (npentrel) [inline docs/configure/_index.md:218]:
```suggestion

{{<gif webm_src="/test.webm" mp4_src="/test.mp4" alt="Test a camera stream" max-width="600px" class="aligncenter">}}

```

Comment (npentrel) [reply docs/configure/_index.md:218]: will work if you rebase (it's the gif from the getting started page)

**Review by npentrel — APPROVED** [review]

**Pull request merged**